### PR TITLE
chore: never mind background voting loop

### PR DESCRIPTION
**Pull request by dckc**
The voting loop is redundant and it occasionally results in:

```
rpc error: code = Unknown desc = account sequence mismatch
```

as in https://github.com/Agoric/agoric-sdk/pull/9250
https://github.com/Agoric/agoric-sdk/actions/runs/8727691415/job/23957208797

### CODE REVIEW

File: contract/scripts/propose-start-contract.sh [hunk 1]
```diff
@@ -20,3 +20,15 @@ agd tx gov submit-proposal swingset-core-eval "$PERMIT" "$SCRIPT" \
 set +x # not so noisy for this part
 . /usr/src/upgrade-test-scripts/env_setup.sh
 voteLatestProposalAndWait
+
+sleep 2
```
Comment (mhofman) [inline contract/scripts/propose-start-contract.sh:24]: What is this sleep working around (I have PTSD anytime I see a delay in tests that is not looking for a trigger)

Comment (dckc) [reply contract/scripts/propose-start-contract.sh:24]: `voteLatestProposalAndWait` finishes when the core-eval proposal passes. This gives it time to execute before we check to see if it works. IOU a comment.

The check was introduced a while ago without this delay, so I think it's not actually essential.

Comment (mhofman) [reply contract/scripts/propose-start-contract.sh:25]: Would it not be better to wait 2 blocks instead?

Comment (dckc) [reply contract/scripts/propose-start-contract.sh]: ok

File: contract/scripts/propose-start-contract.sh [hunk 2]
```diff
@@ -20,3 +20,16 @@ agd tx gov submit-proposal swingset-core-eval "$PERMIT" "$SCRIPT" \
...
+
+# Give the core-eval script a block or two to run.
+waitForBlock 2
+
+parseInstances() {
+  # extract latest value; parse smallCaps; yield 1 line per entry
+  jq -c '.value | fromjson | .values[-1] | fromjson | .body[1:] | fromjson | .[]'
+}
+
+# check that the contract was actually started
```
Comment (mhofman) [inline contract/scripts/propose-start-contract.sh:32]: Hard for me to track this down right now, but will a failure of this check result in the `yarn start:contract` to fail? I don't see any `exit` here if that were the case, but I also don't know of the error code would bubble all the way back.

File: contract/scripts/propose-start-contract.sh [hunk 3]
```diff
@@ -20,3 +20,16 @@ agd tx gov submit-proposal swingset-core-eval "$PERMIT" "$SCRIPT" \
 set +x # not so noisy for this part
 . /usr/src/upgrade-test-scripts/env_setup.sh
 voteLatestProposalAndWait
```
Comment (mhofman) [inline contract/scripts/propose-start-contract.sh:22]: We're we already trying to approve on this side before?

**Review by mhofman — COMMENTED** [review]
Some question about error handling.

File: contract/scripts/propose-start-contract.sh (hunk 3 above)
Comment (dckc) [reply contract/scripts/propose-start-contract.sh:22]: exactly.

File: contract/scripts/propose-start-contract.sh (hunk 2 above)
Comment (dckc) [reply contract/scripts/propose-start-contract.sh:32]: yes, the error code does bubble all the way back

Comment (mhofman) [reply contract/scripts/propose-start-contract.sh:32]: Oh right, if grep doesn't find a match, it's a pipe fail. No need to explicitly exit.

**Review by mhofman — APPROVED** [review]

**Comment by toliaqat** [comment]
> The voting loop is redundant
@dckc how is the voting happening now to approve the proposal?

**Comment by dckc** [comment]
> > The voting loop is redundant
> 
> @dckc how is the voting happening now to approve the proposal?

with `voteLatestProposalAndWait` in `propose-start-contract.sh`, as noted [above](#discussion_r1570902320)

**Pull request merged**